addons/cookoff/XEH_postInit.sqf

```diff
@@ -80,11 +80,9 @@ GVAR(cacheTankDuplicates) = call CBA_fnc_createNamespace;
 // secondary explosions
 ["AllVehicles", "killed", {
     params ["_vehicle"];
-    if (_vehicle getVariable [QGVAR(enable),GVAR(enable)]) then {
```

Comment:
:+1: Support the removal of this dependency, can see cases where mission designer wouldn't want vehicle cook off and the damage reduction associated with it, but still want the ammo cook off on kill.